fflas-ffpack/ffpack/ffpack.h

```diff
@@ -2032,6 +2032,52 @@ namespace FFPACK { /* SSS */
                            typename Field::ConstElement_ptr W, size_t ldw,
                            typename Field::ConstElement_ptr D, size_t ldd,
                            typename Field::Element_ptr A, size_t lda);
+            /**
+         * @brief Computes a quasi-separable matrix A from its SSS generators
```

Comment:
the converse: computes the SSS generators of a dense quasi-separable matrix

Reply:
Fixed in [9dc4531]